stdlib/public/SDK/Foundation/CheckClass.mm

```diff
@@ -1,7 +1,15 @@
 #import <Foundation/Foundation.h>
 
+#include <objc/runtime.h>
+
 #include "swift/Runtime/Metadata.h"
 
+// Aliases for Objective-C runtime entry points.
+static const char *class_getName(const swift::ClassMetadata* type) {
+  return class_getName(
+    reinterpret_cast<Class>(const_cast<swift::ClassMetadata*>(type)));
+}
```

Comment:
You don't need this part; you already have the metadata as an Objective-C `Class`.